server/src/main/java/org/opensearch/cluster/awarenesshealth/ClusterAwarenessHealth.java

```diff
@@ -0,0 +1,388 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster.awarenesshealth;
+
+import org.opensearch.cluster.ClusterState;
+import org.opensearch.cluster.health.ClusterHealthStatus;
+import org.opensearch.cluster.health.ClusterIndexHealth;
+import org.opensearch.cluster.metadata.IndexMetadata;
+import org.opensearch.cluster.metadata.WeightedRoutingMetadata;
+import org.opensearch.cluster.node.DiscoveryNode;
+import org.opensearch.cluster.routing.IndexRoutingTable;
+import org.opensearch.cluster.routing.ShardRouting;
+import org.opensearch.cluster.routing.WeightedRouting;
+import org.opensearch.cluster.routing.allocation.AwarenessReplicaBalance;
+import org.opensearch.cluster.routing.allocation.decider.AwarenessAllocationDecider;
+import org.opensearch.common.collect.ImmutableOpenMap;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Settings;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * Cluster state HA health information
+ *
+ */
+public class ClusterAwarenessHealth implements Writeable {
+
+    private final int numberOfNodes;
+    private final int numberOfDataNodes;
+    private int activeShards;
+    private int unassignedShards;
+    private int initializingShards;
+    private ClusterHealthStatus status;
+    private final Map<String, Object> collatedAttributeHealth;
```

Comment:
Instead of `Object`, lets created `AttributeHealth` class.

Reply:
As discussed we are having multiple attribute value and per attribute value we are returning a list of Attribute values so instead of object class AttributeHealth class cannot be used